index.html

```diff
@@ -1,414 +1,355 @@
+                    <li>2015</li>
                 </ul>
             </div>
-            <p class="content">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
-            <br>
-            <div class="buttons">
-                <ul class="list">
-                    <li><div class="tag">html</div></li>
-                    <li><div class="tag">Ruby on rails</div></li>
-                    <li><div class="tag">css</div></li>
-                    <li><div class="tag">javascript</div></li>
-                </ul>
-                <button>See Project</button>
+            <div class="modal-bottom">
+                <div class="modal-img-conatiner">
+                    <img class="hide-on-lg" src="./images/snapshot-profile-1.png" alt="modal image">
+                    <img class="show-on-lg hide-on-sm" src="./images/card-img1.png" alt="modal image desktop">
+                </div>
+                <div class="wrapper">
+                    <div class="modal-para-conatiner">
+                        <p>
+                            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+                        </p>
+                    </div>
+                    <div class="modal-button-conatiner">
+                        <div class="modal-technologies">
+                            <ul class="list grid-list">
+                                <li><div class="#">html</div></li>
+                                <li><div class="#">css</div></li>
+                                <li><div class="#">javascript</div></li>
+                                <li><div class="#">github</div></li>
+                                <li><div class="#">ruby</div></li>
+                                <li><div class="#">Bootstraps</div></li>
+                            </ul>
+                        </div>
+                        <div class="modal-buttons flex align-center justify-center">
+                            <a href="https://1gordon.github.io/my-portfolio/
+                            " class="button-style modal-btn">See Live <img class="button-icons" src="./images/see-live-icon.svg" alt="see live icon"></a>
+                            <a href="https://github.com/1GORDON/my-portfolio
+                            " class="button-style modal-btn">See Source<img class="button-icons" src="./images/vector-4.png" alt="github icon"></a>
+                        </div>
+                    </div>
+                </div>
             </div>
-        </section>
-    </section>
-</section>
+        </div>
+    </div>
+
+    <!-- project section -->
+    <section class="projects"></section>
```

Comment:
- [ ] Check the blue lines to see what you can improve in your desktop popup design.
- [ ] Add the right background.
- [ ] Your text changes a lot when I try to resize the desktop version, kindly fix it due to sometimes part of your text disappears when I resize the tab.

Your desktop design

![image](https://user-images.githubusercontent.com/60164239/131984555-3675b880-d4c6-4041-9765-d1894e321ea8.png)

Figma desktop design

![image](https://user-images.githubusercontent.com/60164239/131985028-af4ff4d0-01c6-4e1c-8e1e-92c152daa981.png)

Reply:
Thank you for the review. let me make the necessary changes